findfood/dyscover.py

```diff
@@ -0,0 +1,74 @@
+import requests
+
+
+def get_lat_lon(location, header):
+    location_url = "%s%s" % (
+        "https://developers.zomato.com/api/v2.1/locations?query=",
+        location,
+        )
+
+    response = requests.get(location_url, headers=header)
+
+    coordinates = {}
+    try:
+        res = response.json()
+    except ValueError:
+        coordinates["Error"] = "ValueError"
+        return coordinates
+    else:
+        coordinates["latitude"] = (res["location_suggestions"][0]["latitude"])
+        coordinates["longitude"] = (
+            res["location_suggestions"][0]["longitude"]
+            )
+        return coordinates
+
+
+def search_restaurants(location, start="0", queries="5"):
+    results = {}
+
+    header = {
+        "User-agent": "curl/7.43.0",
+        "Accept": "application/json",
+        "user_key": "73b44e74c879ee61cd9b0220830e3971",
+        }
+
+    location = get_lat_lon(location, header)
+
+    if "Error" in location:
+        return results
+
+    lat = str(location["latitude"])
+    lon = str(location["longitude"])
+
+    search_url = "%s%s%s%s%s%s%s%s%s" % (
```

Comment:
this is very unreadable. You should only try to substitute dynamic parts of the string. For example - `https://developers.zomato.com/api/v2.1/search?` doesn't need to be substituted. Your string should be like this - `"https://developers.zomato.com/api/v2.1/search?start=%s..."`